docs/glossary.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Cluster API Scope and Objectives
+authors:
+  - "@dhellmann"
+reviewers:
+  - "@timothysc"
```

Comment:
This should probably be the union of both cluster-api-admins and cluster-api-maintainers here: https://github.com/kubernetes-sigs/cluster-api/blob/master/OWNERS_ALIASES

Reply:
Should I update the authors, reviewers, or both?

Reply:
I think it's fine for the creator of the doc, it's not an OWNERS file.

Reply:
Sorry, just the reviewers :)

Reply:
Done